#!/usr/bin/env python3
"""
MovementData        Define movement for robot      
"""


class MovementData:
    
    def __init__(self):
        self.serialID = 0
        self.driveDirection = 0
        self.stop = 0
        self.packIn = 0
        self.packOut = 0
        self.raiseBucketChain = 0
        self.lowerBucketChain = 0
        self.speedBucketChain = 0
        self.raiseConveyorBelt = 0
        self.lowerConveyorBelt = 0
        self.speedConveyorBelt = 0
        self.misc1 = 0
        self.misc2 = 0
        return


